tests/integration/questionnaire/test_questionnaire_interstitial.py

```diff
@@ -22,7 +22,7 @@ def test_interstitial_can_continue_and_submit(self):
         self.assertInUrl(SUBMIT_URL_PATH)
         self.post()
         self.assertInUrl(THANK_YOU_URL_PATH)
-        self.assertInBody("Submission successful")
+        self.assertInBody("Thank you for completing the Interstitial Pages")
```

Comment:
I don't see what this assert is giving over asserting that the url matches the thank you page, I think we should remove it.

Reply:
Removed the assertion, thanks!